### PR TITLE
doc: garbage collector: detection of containers for deletion

### PR DESCRIPTION
**Pull request by ingvagabund**
Adding description of detection of containers for deletion and steps in which all relevant containers are selected.

### CODE REVIEW

File: admin_guide/garbage_collection.adoc [hunk 1]
```diff
@@ -58,6 +58,20 @@ kubeletArguments:
 ----
 ====
 
+==== Detection of containers for deletion
+
+At the moment only docker and rkt are supported. Rkt has its own GC. The following is only docker related.
+
+First, garbage collector retrieves a list of available containers.
```
Comment (csrwng) [inline admin_guide/garbage_collection.adoc]: First, _the_ garbage collector...

File: admin_guide/garbage_collection.adoc [hunk 2]
```diff
@@ -58,6 +58,20 @@ kubeletArguments:
...
+Filters out all containers that are running or does not live longer then `minimum-container-ttl-duration` parameter.
```
Comment (csrwng) [inline admin_guide/garbage_collection.adoc]: This seems like a continuation of the previous sentence... maybe separate with a ";"

File: admin_guide/garbage_collection.adoc [hunk 3]
```diff
@@ -58,6 +58,20 @@ kubeletArguments:
...
+Then determines pod membership of each remaining container.
+Removes all unidentified containers (with no pod membership).
```
Comment (csrwng) [inline admin_guide/garbage_collection.adoc]: same for these... maybe they should be put in a bulleted list:
The garbage collector
1) retrieves a list of available containers
2) filters out containers... 
3) ...

**Comment by csrwng** [comment]
sorry closed by mistake... my main comment is that instead of having several incomplete sentences, you should use either a bulleted or numbered list to indicate the sequence of events.

**Comment by ncdc** [comment]
@csrwng @ingvagabund  is this still pending updates?

**Comment by csrwng** [comment]
@ncdc @ingvagabund looks like @adellape merged a version of the current doc (without this update) in openshift-docs. 

@ingvagabund would you mind opening this as a separate PR directly on openshift-docs ?

**Comment by adellape** [comment]
Sorry, didn't realize this was here. :/

**Comment by ingvagabund** [comment]
Here https://github.com/openshift/openshift-docs/pull/1219

**Pull request closed without merging**